apps/mymedicare_cb/views.py

```diff
@@ -101,13 +104,25 @@ def callback(request):
     url = fhir_source.fhir_url + \
         "Patient/?identifier=http%3A%2F%2Fbluebutton.cms.hhs.gov%2Fidentifier%23hicnHash%7C" + \
         crosswalk.user_id_hash + \
-        "&_format=json"
-    response = requests.get(url, cert=certs, verify=False)
+        "&_format=application%2Fjson%2Bfhir"
+    # We may need to grab the current headers and pass the dict
+    # to set_header_content_type()
+    response = requests.get(url,
+                            cert=certs,
+                            verify=False)
 
     if 'entry' in response.json() and response.json()['total'] == 1:
         fhir_id = response.json()['entry'][0]['resource']['id']
         crosswalk.fhir_id = fhir_id
         crosswalk.save()
+        logger.INFO("Crosswalk entry created "
+                    "for Beneficiary:%s[len:%s]" % (user.username,
+                                                    len(fhir_id)))
```

Comment:
I'm not sure I'm comfortable adding this information to our logs. This is clear PII